ui2.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="author" content="Ozan Paker">
+    <meta name="keywords" content="html pratikleri,html">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML-Practices</title>
+    <!-- title ne işe yarar? -->
+    <link id="favicon" rel="shortcut icon" type="image/png" href="favicon.png"/>
+  </head>
+  <body>
+    <h1>html öğrenme</h1>
+    <h2>html öğrenme</h2>
+    <h3>html öğrenme</h3>
+    <h4>html öğrenme</h4>
+    <h5>html öğrenme</h5>
+    <h6>html öğrenme</h6>
+    <hr/>
+    <pre>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+       Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+
+        It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+
+        It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</pre>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+
+          Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+    <hr/>
+    <i>Hebele Hübele</i>
+    <br/>
+    <b>Hebele Hübele</b>
```

Comment:
İtalic için i ve em, kalınlık için de b be strong aynı işi yapıyor biz bunları alt alta da görsek güzel olurdu sanki ne dersn 😇